india_compliance/gst_india/utils/test_e_invoice.py

```diff
@@ -34,6 +34,8 @@ def setUpClass(cls):
                 "auto_generate_e_invoice": 0,
                 "enable_e_waybill": 1,
                 "fetch_e_waybill_data": 0,
+                "apply_e_invoice_only_for_selected_companies": 0,
+                "e_invoice_applicable_from": frappe.utils.today(),
```

Comment:
This was today by default (hence wasn't read). The same can be done child table.

Reply:
you meant to set `default` property in GST Settings?

Reply:
Added default value in child only because default GST settings is already been applied at time of installation.